back/engines/commercial/analysis/spec/acceptance/analyses_spec.rb

```diff
@@ -0,0 +1,112 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require 'rspec_api_documentation/dsl'
+
+resource 'Analyses' do
+  explanation 'Analyses are the parent resources to contain textual, AI assisted sensemaking of inputs'
+
+  before do
+    header 'Content-Type', 'application/json'
+    admin_header_token
+  end
+
+  get 'web_api/v1/analyses' do
+    parameter :project_id, 'Only returns analyses scoped to the project (use for ideation-like methods)', required: false
+    parameter :phase_id, 'Only returns anlyses scoped to the phase (use for survey-like methods)', required: false
+
+    with_options scope: :page do
+      parameter :number, 'Page number'
+      parameter :size, 'Number of analyses per page'
+    end
+
+    before do
+      @survey_analysis = create(:survey_analysis)
+      @ideation_analysis = create(:ideation_analysis)
+    end
+
+    example_request 'List all analyses' do
+      expect(status).to eq(200)
```

Comment:
Just so you know: Today we have a helper method so you can write `assert_status 200`

Reply:
Noted :)